pkg/errutils/errutils.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package errutils
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
+	ErrInvalidFileExtension     = "unsupported file extension: %s. Supported extension(s): %s"
+	ErrInvalidCmdLineArg        = "expected only one argument specifying the input file"
+	ErrFileNotExist             = "file %s does not exist"
```

Comment:
Do we actually check for the existence of the file or is it possible that "read file permissions" are missing.
In that case "failed to open file %s" would be more appropriate.

Reply:
[Addressed review comments and Revised error messages](https://github.com/Open-CMSIS-Pack/cbuild/pull/243/commits/efe32d61b8c0b2faed1818051ccdf45aba4bd29e)